haiku-apps/nanodot/nanodot-1.1.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Binary & source for a simple MIDI sequencer"
+DESCRIPTION="\
+              nanodot is a rather minimal pattern-based MIDI\n\
+               sequencer for BeOS R3. Its main purpose in life\n\
+               was/is to teach me (and hopefully others) the\n\
+               basics of writing MIDI apps for BeOS; it makes\n\
+               for a nice drum-programming 'scratchpad', and is a\n\
+               generally entertaining way to waste a few hours."
+HOMEPAGE="https://github.com/HaikuArchives/NanoDot"
+COPYRIGHT="1999 Eric Moon"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="git+https://github.com/HaikuArchives/NanoDot"
```

Comment:
```diff
 REVISION="1"
+srcGitRev="80cbeedb3f86ddad8a92bfcf7212704f1528da0f"
-SOURCE_URI="git+https://github.com/HaikuArchives/NanoDot"
+SOURCE_URI="https://github.com/HaikuArchives/NanoDot/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="6cd316390ec1939f7b8e21b1495ec9fffccc3d859b8d0caddb5ca6b6b56399cc"
+SOURCE_FILENAME="NanoDot-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="NanoDot-$srcGitRev"
+PATCHES="nanodot-$portVersion.patchset"
```

BTW, since the real release name is "1.1b" (and not "1.1"), it might be better to rename the recipe to `nanodot-1.1b.recipe` and the patch to `nanodot-1.1b.patchset`.
But let's see what the other mentors and/or key contributors think about this.